plugins/inputs/redis_sentinel/redis_sentinel.go

```diff
@@ -0,0 +1,464 @@
+package redis_sentinel
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-redis/redis"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type RedisSentinel struct {
+	Servers  []string `toml:"servers"`
+	Password string   `toml:"password"`
+	tls.ClientConfig
+
+	clients []*RedisSentinelClient
+}
+
+type RedisSentinelClient struct {
+	sentinel *redis.SentinelClient
+	tags     map[string]string
+}
+
+const measurementMasters = "redis_sentinel_masters"
+const measurementSentinel = "redis_sentinel"
+const measurementSentinels = "redis_sentinel_sentinels"
+const measurementReplicas = "redis_sentinel_replicas"
+
+func init() {
+	inputs.Add("redis_sentinel", func() telegraf.Input {
+		return &RedisSentinel{}
+	})
+}
+
+func (r *RedisSentinel) SampleConfig() string {
+	return `
+  ## specify servers via a url matching:
+  ##  [protocol://][:password]@address[:port]
+  ##  e.g.
+  ##    tcp://localhost:26379
+  ##    tcp://:password@192.168.99.100
+  ##    unix:///var/run/redis-sentinel.sock
+  ##
+  ## If no servers are specified, then localhost is used as the host.
+  ## If no port is specified, 26379 is used
+  servers = ["tcp://localhost:26379"]
```

Comment:
As you handle the default case you can use
```suggestion
  # servers = ["tcp://localhost:26379"]
```
here.

Reply:
@srebhan Sure. I also addressed all the lint errors in markdown that popped up on CI. The changes were made in https://github.com/influxdata/telegraf/pull/10042/commits/e8f0286f32fbe780a9c2c1064f686a88a4289f35.